tests/async.rs

```diff
@@ -93,24 +93,35 @@ where
 
     fn test<C, F>(self, c: C)
     where
-        F: IntoFuture<Item = (), Error = mysql_async::errors::Error>,
-        C: FnOnce(mysql_async::Conn) -> F,
+        F: IntoFuture<Item = (), Error = mysql_async::error::Error> + 'static,
+        <F as futures::IntoFuture>::Future: std::marker::Send,
+        C: FnOnce(mysql_async::Conn) -> F + Send + 'static,
     {
+        let mut runtime = Runtime::new().unwrap();
```

Comment:
I think you could get away without the extra bounds above by using [`current_thread::Runtime`](https://docs.rs/tokio/0.1/tokio/runtime/current_thread/index.html) instead. It will also not spin up a ton of threads just for testing, so that might be better anyway :)

Reply:
oh yeah, good call.